tests/support/isGetFocusedGranuleQuery.js

```diff
@@ -0,0 +1,14 @@
+const variablesMatch = (variables, granuleId) => variables.params.conceptId === granuleId
+
+export const isGetFocusedGranuleQuery = (route, granuleId) => {
```

Comment:
Might we worth while to do some jsdoc for these helper functions

Reply:
:doh: Definitely want that